test/Microsoft.AspNetCore.CookiePolicy.Test/CookieConsentTests.cs

```diff
@@ -221,6 +221,8 @@ public class CookieConsentTests
             var consentCookie = cookies[0];
             Assert.Equal(".AspNet.Consent", consentCookie.Name);
             Assert.Equal("yes", consentCookie.Value);
+            Assert.True(consentCookie.Expires.HasValue);
+            Assert.True(consentCookie.Expires.Value > DateTimeOffset.Now + TimeSpan.FromDays(360));
```

Comment:
Nit: why is this checking only 360 days from now?

Reply:
365 would be cutting the comparison too close. 364?